src/main/kotlin/ua/marchenko/artauction/common/extension/Enum.kt

```diff
@@ -0,0 +1,3 @@
+package ua.marchenko.artauction.common.extension
+
+inline fun <reified T : Enum<T>> valueOfOrNull(name: String): T? = runCatching { enumValueOf<T>(name) }.getOrNull()
```

Comment:
```suggestion
inline fun <reified T : Enum<T>> safeValueOfOrNull(name: String): T? = runCatching { enumValueOf<T>(name) }.getOrNull()
```

from method name it's not clear that exceptions are suppressed

Reply:
it is a method like `EnumClass.valueOf(value: String): EnumClass`, but returns `null` instead of throwing `IllegalArgumentException`. I have named it like this due to Kotlin method naming like `toIntOrNull()`, `getOrNull()` etc